pom.xml

```diff
@@ -827,6 +851,9 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>2.10.1</version>
+          <configuration>
+            <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
Add a comment to remove this after fixing all the java doc issues.

Reply:
done